golem/payload/defaults.py

```diff
@@ -14,14 +14,20 @@
 
 @dataclass
 class NodeInfo(Payload):
-    """Properties describing the information regarding the node."""
+    """Properties and constraints describing the information regarding the node."""
 
     name: Optional[str] = prop("golem.node.id.name", default=None)
     """human-readable name of the Golem node"""
 
     subnet_tag: Optional[str] = prop("golem.node.debug.subnet", default=None)
+    _subnet_tag_constraint: Optional[str] = constraint(
```

Comment:
on the other hand, after today's discussion, I'm wondering whether the _subnet_tag_ shouldn't rather be part of the payload - especially given that we might wish to vary it between the providers... but maybe for now, let's leave it here...